todo_app/app.py

```diff
@@ -1,14 +1,34 @@
-from flask import Flask
+from flask import Flask, render_template, request, redirect, url_for
+import todo_app.data.session_items as session
 
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
-
-@app.route('/')
+@app.route('/', methods=['GET', 'POST'])
 def index():
-    return 'Hello World!'
+    if request.method == 'GET':
+        items = session.get_items()
+        return render_template('index.html', myitems=items)
+    elif request.method == 'POST':
+        #Add new item
+        if request.form.get('title'):
+            title = request.form.get('title')
+            session.add_item(title)
+        #Delete item
+        if request.form.get('del_id'):
+            del_id = int(request.form.get('del_id')) if request.form.get('del_id') else 0
```

Comment:
`if request.form.get('del_id') else 0` doesn't do anything - you're already inside an `if request.form.get('del_id')`